x/ratelimit/types/msgs.go

```diff
@@ -41,7 +42,14 @@ func (msg *MsgAddQuota) GetSignBytes() []byte {
 }
 
 func (msg *MsgAddQuota) ValidateBasic() error {
```

Comment:
can you add unit tests for these `ValidateBasic` functions?

Reply:
ok